spec/puppet-server-lib/puppet/jvm/ast_compiler_spec.rb

```diff
@@ -25,12 +25,17 @@ def request(code)
    "variables" => {"values" => {"foo" => "bar"}}}
 end
 
-def find_notify(catalog)
+def find_notify(catalog, title: nil)
   catalog['resources'].find do |item|
-    item['type'] == 'Notify'
+    next item['type'] == 'Notify' unless title
+    item['type'] == 'Notify' && item['title'] == title
   end
 end
 
+def collision_warning(msg_regex)
```

Comment:
I dont see where this is used?

Reply:
This was from a previous version of the unit tests when I was looking through the logs. Removed.